public/css/index.css

```diff
@@ -106,7 +106,7 @@ body.night{
     color: #78B2F2 !important;
 }
 .CodeMirror-sizer {
-    margin-bottom: 0px !important;
+    margin-bottom: 26px !important;
```

Comment:
magic number! This needs either a comment what the 26px refer to.

Reply:
Git blame would tell, but let me add a comment here :)

Reply:
In that direction, yes. But I'm thinking the other way round: If someone were to modify the status bar, they would never find `.CodeMirror-sizer` or "all things 26px". Coming from that other angle, git blame won't help.